packages/types/src/clerk.ts

```diff
@@ -104,6 +104,8 @@ export interface Clerk {
    */
   loaded: boolean;
 
+  __internal_getOption<K extends keyof ClerkOptions>(key: K): ClerkOptions[K];
```

Comment:
I'd suggest to place this as a helper outside of the main `Clerk` interface as we're using this mostly for typing purposes.

Reply:
@LauraBeatris can you share more? I am not completely following the change requested.

Reply:
Ohhh, I'm sorry! Reading the code at first, I thought it was only an internal utility method for accessing a property against `ClerkOptions` keys - you can ignore that one.